.cicd/platforms/pinned/macos-10.15-pinned.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
I assume this file is very similar to the Mojave file?

Reply:
Yep